cloudhub-2/modules/ROOT/pages/ch2-deploy-api.adoc

```diff
@@ -0,0 +1,180 @@
+= Deploy an App with TCP using the Application Manager API
+
+To configure protocols other than HTTP or HTTPS, configure TCP inbound rules to manage traffic for your private space. You can then use the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/amc-application-manager/minor/4.0/console/method/%231158/[Application Manager API] to deploy an app using TCP. 
```

Comment:
Consider "set up" for the second "configure" to avoid repetition

Reply:
```suggestion
To configure protocols other than HTTP or HTTPS, set up TCP inbound rules to manage traffic for your private space. You can then use the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/amc-application-manager/minor/4.0/console/method/%231158/[Application Manager API] to deploy an app using TCP. 
```